src/utils/__tests__/prepareBlocks.test.js

```diff
@@ -64,4 +64,59 @@ describe('createBlocks', () => {
       }).preparedBlocks,
     ).toHaveLength(3);
   });
+
+  it('should create empty release for teams with channels when there are no messages', () => {
+    const exampleTeam = Team({
+      color: 'blue',
+      emoji: '🤷‍♀️',
+      mentions: '@MillhouseManaStorm',
+      name: 'FRIENDS',
+      blocks: Array(5).fill([createStorySection('hi')]),
+      issueTracking: {
+        jira: {
+          projects: ['CHANDLER', 'MONICA'],
+        },
+      },
+      channels: ['test-channel-1', 'test-channel-2'],
+    });
+    const blocks = createBlocks(false, {
```

Comment:
got confused on `false` here ... maybe we can set a variable with a more helpful name to pass in instead.  Sounds like this is where/how we provide "no messages" to `createBlocks`

Reply:
fixed with https://github.com/target/captains-log/pull/157/commits/ca01e4905ba426833a8fc51823ab7d047e3e5f2f - using the same variable name as the function uses `hasMessages`